packages/core/package.json

```diff
@@ -4,6 +4,12 @@
   "license": "MIT",
   "type": "module",
   "dependencies": {
+    "@anthropic-ai/sdk": "^0.9.1",
+    "@gxl/epub-parser": "2.0.4",
+    "@notionhq/client": "^2.2.13",
+    "@xenova/transformers": "^2.8.0",
+    "crypto-js": "^4.2.0",
+    "js-tiktoken": "^1.0.8",
```

Comment:
this should only add the epub parser

Reply:
Yes, I'm still working on this. Sorry for bothering you.
Will change this pr to draft until it's ready.